motion/dp_guidance/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>dp_guidance</name>
+  <version>0.0.0</version>
+  <description>The dp_guidance package</description>
+
+  <maintainer email="chstr@stud.ntnu.no">christopher</maintainer>
+  <license>MIT</license>
+  
```

Comment:
Totally wrong, Christopher! You have to leave it as <maintainer email="TODO@TODO"> ;)

Reply:
And name as S. Anntid? :thinking: